PR: https://github.com/odoo/odoo/pull/

From: 98748a93c083307e1f853c820725c7807ff86135
From: Deep Patel
Date: 2017-09-29 12:12:03

Structural Changes: 6
Total Changes: 8

[FIX] res_config: Fix several layout issues on config views

- subscription settings are broken and show up at the bottom of any setting page: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrra01leUZ1Wnczbms/view?usp=drivesdk
- don't put setting item of the app you are configuring at the top of the list: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrdXEwUmFWTTlLd1E/view?usp=drivesdk
- hide empty sections when searching not in debug mode: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrVGFOa0FKVmtNTWs/view?usp=drivesdk
- when installing Invoicing, setting item should be Invoicing and not "Accounting": https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrWGQwY1ktaXlNSUk/view?usp=drivesdk
- app labels: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrreXlNTXZnbU13aFk/view?usp=drivesdk
- coupon programs links: duplicate in sales + no link at all in website: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrraTV1YTNPdWU2VUE/view?usp=drivesdk
- delivery methods: links are duplicated in sales settings https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrczR5X1FOUllUZHM/view?usp=drivesdk
- add subtitles to all the carriers in sales settings: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrS1NXXzhwRTFQV2c/view?usp=drivesdk
- inner options should never be in bold:
	customer account: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrMnpualBta2x6dU0/view?usp=drivesdk
	phone validation: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrraHJrYngtaGJ6T2s/view?usp=drivesdk
	signature & payment: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrWTJ3dVBuTzNONlU/view?usp=drivesdk
	bill control: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrRy16UkdTWWdrdDQ/view?usp=drivesdk
- make similar settings look the same through all the apps:
	taxcloud: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrraGlVWmNXRjJ5WEk/view?usp=drivesdk
	pricelist: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrQ2pwVmtEZXM1SWM/view?usp=drivesdk
	multi-currencies: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrTFNWTTJpcS1XbnM/view?usp=drivesdk
	shipping costs: https://drive.google.com/a/odoo.com/file/d/0B1uIL9E_zXrrM2hKdk5XRy1WTjg/view?usp=drivesdk

================================= pseudo patch: =================================

--- a/addons/sale/models/res_config_settings.py
+++ b/addons/sale/models/res_config_settings.py
@@ -22,10 +22,10 @@ class ResConfigSettings(models.TransientModel):
     module_sale_payment = fields.Boolean("Online Signature & Payment", help='Technical field implied by user choice of online_confirmation')
     module_website_quote = fields.Boolean("Quotations Templates")
     group_sale_delivery_address = fields.Boolean("Customer Addresses", implied_group='sale.group_delivery_invoice_address')
-    multi_sales_price = fields.Boolean("Multiple sales price per product")
+    multi_sales_price = fields.Boolean("Multiple Sales Prices per Product")
     multi_sales_price_method = fields.Selection([
         ('percentage', 'Multiple prices per product (e.g. customer segments, currencies)'),
-        ('formula', 'Price computed from formulas (discounts, margins, roundings)')
+        ('formula', 'Prices computed from formulas (discounts, margins, roundings)')
         ], default='percentage', string="Pricelists")
     sale_pricelist_setting = fields.Selection([
         ('fixed', 'A single sales price per product'),
@@ -66,8 +66,8 @@ class ResConfigSettings(models.TransientModel):
         ('b2c', 'Free sign up (B2C)'),
     ], string='Customer Account')
 
-    module_delivery = fields.Boolean("Manage shipping internally")
-    module_delivery_dhl = fields.Boolean("DHL USA")
+    module_delivery = fields.Boolean("Shipping Costs")
+    module_delivery_dhl = fields.Boolean("DHL")
     module_delivery_fedex = fields.Boolean("FedEx")
     module_delivery_ups = fields.Boolean("UPS")
     module_delivery_usps = fields.Boolean("USPS")
